vite build types error

![image](https://user-images.githubusercontent.com/35677796/178947040-b143e9e7-81ba-4b72-9e34-73949c16869c.png) vite 
Error while building


**node_modules/swr/_internal/dist/_internal/utils/env.d.ts:4:76 - error TS2694: Namespace 'global.NodeJS' has no exported member 'Timeout'.

4 export declare const rAF: (f: (...args: any[]) => void) => number | NodeJS.Timeout;**

--- a/_internal/utils/env.ts
+++ b/_internal/utils/env.ts
@@ -6,7 +6,9 @@ export const IS_REACT_LEGACY = !React.useId
 export const IS_SERVER = !isWindowDefined || 'Deno' in window
 
 // Polyfill requestAnimationFrame
-export const rAF = (f: (...args: any[]) => void) =>
+export const rAF = (
+  f: (...args: any[]) => void
+): number | ReturnType<typeof setTimeout> =>
   hasRequestAnimationFrame()
     ? window['requestAnimationFrame'](f)
     : setTimeout(f, 1)

--- a/test/type/internal.tsx
+++ b/test/type/internal.tsx
@@ -1,0 +1,8 @@
+import { rAF } from 'swr/_internal'
+import { expectType } from './utils'
+
+export function rAFTyping() {
+  expectType<
+    (f: (...args: any[]) => void) => ReturnType<typeof setTimeout> | number
+  >(rAF)
+}

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -24,7 +24,6 @@
       "swr/immutable": ["./immutable/index.ts"],
       "swr/mutation": ["./mutation/index.ts"]
     },
-    "typeRoots": ["./node_modules/@types"],
     "incremental": true
   },
   "include": ["core"],

